prisma/schema.prisma

```diff
@@ -355,6 +355,7 @@ model Asset {
   name                   String
   owner                  String
   supply                 BigInt
+  verified               Boolean            @default(false)
```

Comment:
nit: This is personal preference, but I opt for nullable timestamps instead of booleans if it makes sense (i.e. using `verified_at` instead of `verified`). If there's ever a need to see when an asset was verified, we'll have it and `verified` can be inferred if it's not `NULL`.

Reply:
Sgtm